pages/home.jsx

```diff
@@ -101,13 +105,15 @@ var BlogSection = React.createClass({
     }
   },
   componentDidMount: function() {
-    var self = this;
-    blogFeedLoader(this, window, "https://blog.webmaker.org/tag/teachtheweb/feed", function(data) {
-      self.setState({
-        featuredPostData: data.featuredPostData,
-        latestPostsData: data.latestPostsData
+    this.props.loadBlogPosts(function(data) {
+      if (!this.isMounted()) {
+        return;
+      }
+      this.setState({
+        featuredPostData: data.featuredPosts,
+        latestPostsData: data.latestPosts
       });
-    });
+    }.bind(this));
```

Comment:
I generally prefer using `.bind()` to defining `var self = this`, partly because it helps ensure that I don't over-nest stuff. (Seeing `self` a bunch of times doesn't bother me, but seeing a ton of `.bind(this)`'s tells me I might be over-nesting. I guess maybe it's just a stylistic thing though. :grin:)